### DISCUSSION TITLE
Next Release

### DISCUSSION THREAD

**Discussion by luzip665** [exaile/exaile#943] [General]
Hi all,

after a year I would like to release Exaile 4.1.4.

Are there any objections?
Are there PRs to wait for?

Regards,
André

**Comment by sjohannes** [comment]
Hey, thanks for this. No issues from me, but I haven't been following the recent bugs/PRs.

I've regenerated the translations file. If you make a beta or RC release that would give translators some time to add the new translations.

**Comment by TheTumultuousUnicornOfDarkness** [comment]
@luzip665 any plan about final v4.1.4 release?

**Comment by luzip665** [comment]
Sorry, I missed.

I can do the release next week. 

@virtuald @sjohannes Do you have an objection?

**Comment by luzip665** [comment]
Done.

**Comment by sjohannes** [comment]
@luzip665 Thanks for that! Sorry I didn't reply earlier.
